guides/v2.1/cloud/env/smart-wizards.md

```diff
@@ -11,32 +11,75 @@ functional_areas:
 
 The smart wizards can assist you in determining if your Cloud configuration follows best practices by verifying the following:
 
--  Enabled SCD on-demand
--  Enabled SCD for the build stage
--  Enabled SCD for the deploy stage
-<!-- -  Enabled the master-slave connections for database and Redis service configuration -->
+-  Ideal state for minimal downtime
+-  Enabled SCD for on-demand, the build stage, or the deploy stage
```

Comment:
"on-demand" what? Should we spell out SCD for people? Is one of these three preferred? I'm struggling with the wording.